packages/governance/test/unitTests/test-typedParamManager.js

```diff
@@ -0,0 +1,336 @@
+  t.is(invitationActualAmount[0].description, 'simple');
+
+  t.is(await paramManager.getInternalParamValue('Invite'), invitation);
+
+  t.deepEqual(
+    paramManager.getParams(),
+    harden({
+      Amt: {
+        type: ParamType.AMOUNT,
+        value: drachmaAmount,
+      },
+      Currency: {
+        type: ParamType.BRAND,
+        value: drachmaBrand,
+      },
+      Invite: {
+        type: ParamType.INVITATION,
+        value: invitationAmount,
+      },
+    }),
+  );
+});
+
+test('two Nats', async t => {
+  const paramManager = makeParamManagerSync({
+    Acres: ['nat', 50n],
+    SpeedLimit: ['nat', 299_792_458n],
+  });
+
+  t.is(paramManager.getAcres(), 50n);
+  t.is(paramManager.getSpeedLimit(), 299_792_458n);
+
+  // @ts-expect-error
+  t.throws(() => paramManager.updateSpeedLimit(300000000), {
+    message: '300000000 must be a bigint',
+  });
+
+  t.throws(() => paramManager.updateSpeedLimit(-37n), {
+    message: '-37 is negative',
+  });
+});
+
+test('Ratio', async t => {
+  const unitlessBrand = makeIssuerKit('unitless').brand;
+
+  const ratio = makeRatio(16180n, unitlessBrand, 10_000n);
+  const paramManager = makeParamManagerSync({
+    Acres: ['nat', 50n],
+    GoldenRatio: ['ratio', ratio],
+  });
+  t.is(paramManager.getGoldenRatio(), ratio);
+
+  const morePrecise = makeRatio(1618033n, unitlessBrand, 1_000_000n);
+  paramManager.updateGoldenRatio(morePrecise);
+  t.is(paramManager.getGoldenRatio(), morePrecise);
+  // @ts-expect-error
+  t.throws(() => paramManager.updateGoldenRatio(300000000), {
+    message: '"ratio" 300000000 must be a pass-by-copy record, not "number"',
+  });
+});
+
+test('Branded Ratio', async t => {
+  const unitlessBrand = makeIssuerKit('unitless').brand;
+
+  const ratio = makeRatio(16180n, unitlessBrand, 10_000n);
+  const paramManager = makeParamManagerSync({
+    Acres: ['nat', 50n],
+    GoldenRatio: ['brandedRatio', ratio],
+  });
+  t.is(paramManager.getGoldenRatio(), ratio);
+
+  const morePrecise = makeRatio(1618033n, unitlessBrand, 1_000_000n);
+  paramManager.updateGoldenRatio(morePrecise);
+  t.is(paramManager.getGoldenRatio(), morePrecise);
+
+  const anotherBrand = makeIssuerKit('arbitrary').brand;
+
+  t.throws(
+    () =>
+      paramManager.updateGoldenRatio(makeRatio(16180n, anotherBrand, 10_000n)),
+    {
+      message:
+        'Numerator brand for "GoldenRatio" must be "[Alleged: unitless brand]"',
+    },
+  );
+});
+
+test('Strings', async t => {
+  const paramManager = makeParamManagerSync({
+    Acres: ['nat', 50n],
+    OurWeapons: ['string', 'fear'],
+  });
+  t.is(paramManager.getOurWeapons(), 'fear');
+
+  paramManager.updateOurWeapons('fear,surprise');
+  t.is(paramManager.getOurWeapons(), 'fear,surprise');
+  // @ts-expect-error
+  t.throws(() => paramManager.updateOurWeapons(300000000), {
+    message: '300000000 must be a string',
+  });
+});
+
+test('Unknown', async t => {
+  const paramManager = makeParamManagerSync({
+    Label: ['string', 'birtday'],
```

Comment:
```suggestion
    Label: ['string', 'birthday'],
```
... unless this was intentional?

Reply:
just a typo, fixed